.storybook/stories/CardsWithRemoteData.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef } from 'react';
```

Comment:
This is a new storybook example of remote data fetching + `Card` components + `Skeleton` components